bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/form/ReservedProperties.java

```diff
@@ -158,6 +158,16 @@ private ReservedProperties() {
     public static final String PN_EXCLUDE_FROM_DOR = "excludeFromDor";
     public static final String PN_MANDATORY = "mandatory";
     public static final String PN_HTML_ELEMENT_TYPE_V2 = "fd:htmlelementType";
+
+    public static final String PN_SUBMIT_ACTION_TYPE = "actionType";
```

Comment:
Please group all submission-related properties together, preferably at the end, and wrap the code with comments indicating the start and end of submission properties.

Reply:
Done